### DISCUSSION TITLE
Who's hiring? (Mar 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#34949] [Ideas]
# What is this?
This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

*If you're looking for last month's discussion, see #33869. At least one company has [hired someone](https://twitter.com/Nutlope/status/1475929984174895105) as a result of it!* 

If you got hire someone through this job board, please reach out to me (hassan@vercel.com) so I can send you some swag :)

If you're a developer looking to post about themselves, use #34948.

**Comment by dfeldman0** [comment]
Excited to share an opportunity as the technical lead our NextJS / Jamstack web team in the Boston area [Apply Here](https://www.takedajobs.com/job/lexington/jamstack-development-lead/1113/23001036944) . 
Very competitive base salary plus annual and long-term incentive bonuses.

At Takeda, we are transforming the pharmaceutical industry through our R&D-driven market leadership and being a values-led company. To do this, we empower our people to realize their potential through life-changing work. Certified as a Global Top Employer, we offer stimulating careers, encourage innovation, and strive for excellence in everything we do. We foster an inclusive, collaborative workplace, in which our global teams are united by an unwavering commitment to deliver Better Health and a Brighter Future to people around the world.

Lead the development of templates and self-service capabilities to enable the business to rapidly create flexible websites using NextJS
Deep development experience on ReactJS, Next.js with Headless CMS (ex. Sanity, Contentful, ContentStack)
Experience integrating headless CMS, static site generators, and rich site editor (ex Stackbit)
Typescript & ES6 Ninja
Client-side security with OAuth / OIDC-
Experience with search in Jamstack architectures (ex. Algolia, Elastic)
Expert Performance and SEO optimization expertise
Hands on experience with dynamic personalization and behavior-based persona detection
Jamstack DevOps experience (ex. Cloudflare Pages, Netlify, Vercel)
Experience with Edge APIs (ex Cloudfare Workers)

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for June is up (https://github.com/vercel/next.js/discussions/37379) and you're more than welcome to repost this message there!

**Comment by jaxomlotus** [comment]
BurnerPage | Front-End Engineer | NYC or Remote (US only) | Full-time | $115-175k base + benefits + generous equity | [Link](https://burnerpage.com/jobs/3/front-end-engineer)

[BurnerPage](https://burnerpage.com/) is a stealth startup founded by a repeat entrepreneur who sold his last company to Adobe. We are VC backed and looking to make hires 2-5 as we bring our product to market. The above link is for [front-end devs](https://burnerpage.com/jobs/3/front-end-engineer), but we're looking for [backend](https://burnerpage.com/jobs/2/back-end-engineer) / fullstack and [machine learning engineers](https://burnerpage.com/jobs/1/machine-learning-engineer) as well (with different base pay for those positions depending on experience).

BurnerPage is building the future of webpages, harnessing the power of AI and Machine Learning to automatically build webpages and tailor the viewer experience to the individual. [Note: The [current product](https://burnerpage.com/) does not yet advertise this purpose - it was just a manual prototype we launched that hit [#1 on ProductHunt](https://www.producthunt.com/posts/burner-page), but the stealth AI based product is the real focus that we're hiring for].I'm the founder and looking to find amazing folks who love working in Next.js. Feel free to reach out to me directly to apply at avi -(a)- burnerpage.com.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for June is up (https://github.com/vercel/next.js/discussions/37379) and you're more than welcome to repost this message there!

**Comment by jalada** [comment]
**[Pixie Labs](https://pixielabs.io) | Full Stack Developer (Senior Engineer) | Remote (UK-based) | Full-time | See job spec for compensation details | [Link](https://apply.workable.com/pixie-labs/j/18B26E12B9/)**

Pixie Labs is a digital product agency. We partner with organisations to design, build and launch software that brings their most valuable technology-driven experiences to life. As a senior engineer, you’ll help head up our clients’ development projects and write high-quality, reusable code whilst also sharing your knowledge and expertise with the rest of the team to help them do the same.

You may think an agency isn't for you - often it means hitting unrealistic deadlines imposed by clients, or writing terrible throwaway code just so the company gets paid. We're not like that; we focus on delivering at a sustainable pace and working closely with our clients to manage expectations. You can see for yourself how we manage and deliver projects in our [handbook](https://handbook.pixielabs.io/#/).

We're fairly polyglot, but specialise in and make heavy use of Ruby on Rails, Turbo (Hotwire and Stimulus), React and of course Next.js.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for June is up (https://github.com/vercel/next.js/discussions/37379) and you're more than welcome to repost this message there!

**Comment by fredericoo** [comment]
**SaltPay | Midweight/Senior Full-Stack Engineer | Hybrid (Lisbon, Porto, London) | Full-time | Competitive salary | e-mail me! frederico.batista@saltpay.co**

SaltPay is a fintech that aims to help small and medium businesses by providing a diverse set of tools to improve productivity, profit and automate processes, granting them the ability to compete with the big players. It is a very much back-end focussed company that could highly benefit from full-stack or front-end devs that have a keen eye for detail.

I've worked for the company for 1 year and have progressed significantly within it, going from a junior front-end dev to approaching senior full stack abilities. I've also had the chance to shape one of the company's products and feel like my opinions and ideas are always heard. Join me!

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for June is up (https://github.com/vercel/next.js/discussions/37379) and you're more than welcome to repost this message there!

**Comment by goodgravy** [comment]
**[Ought](https://ought.org/)** | **Frontend engineer** | **Remote, or San Francisco** | Full-time | Competitive salary | [Link](https://ought.org/careers/frontend-engineer)

We are using artificial intelligence to help researchers be more effective. We think AI is the keystone technology which will accelerate scientific progress, but we're also profoundly concerned by the risks transformative AI brings with it.

Out current focus – a GPT-3 powered research assistant named [Elicit](https://elicit.org/) – is our proposal for how to capture the upside of artificial intelligence, while building the system in such a way as to mitigate the risks. [Quick demo](https://www.youtube.com/watch?v=rjOHcfU-XvA).

This is an extremely challenging frontend problem, because our goal is to distil insights from hundreds of millions of research papers into a straightforward interface 🙃 . We need excellent engineers to help with this! Design sense is a big plus (although we do have a designer on the team too); ML experience is not required.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for June is up (https://github.com/vercel/next.js/discussions/37379) and you're more than welcome to repost this message there!

**Comment by trun222** [comment]
[Glossi](https://www.glossi.io) | Senior Frontend Developer | Remote | Full-time | Competitive salary | USA Only

# Our Company
Glossi has set out to empower everyone, from amateurs to professionals, entrepreneurs to big brands, to create gorgeous photos and videos of their products in a fraction of the time it traditionally takes (weeks to minutes). We’re building a creative studio-in-a-box that dramatically simplifies the 3D rendering content creation process, and even makes it fun! Think Canva, but 3D. We are creating content for everyone from Kickstarters to Fortune 500 companies. With our 3D model management system, virtual photo studio, and video templates, we eliminate the need for agencies, freelancers, and the outsourcing of work which enables our clients to create everything they need to sell across socials, storefronts, and more. Come join us in defining the future of visual marketing!

# Our Squad
- We enjoy using the latest Saas, libaries, and frameworks where it makes sense
- We have a collaborative work environment where we brainstorm and plan together
- We help each other accomplish our team and company goals
- Growing each team member's career goals and aspirations is important to us
- It is important to us to have a laid back culture
- Feedback is always welcome at all stages of the SDLC

# Responsibilities
- You will build and maintain beautiful, snappy, and native-feeling UI experiences across all of our product teams from video creation to enterprise content management features.
- You will optimize and set new standards for application-level tooling to measure product stability and speed.
- You will build product features end-to-end: through conception, exploration, iteration, maintenance, and polish.
- You will own the features that you build and become the domain expert on those areas within the organization in time.
- You will help collaborate and guide feature sets with UI/UX and Glossi leadership.
- You will help with interviewing other talented engineers that want to join Glossi.
- Manage a high output team through mentorship and teaching other developers.
- Identify ways to improve and guide company planning and design decisions.
- E2E ownership and resolution of bugs
- Clearly communicate with peers and drive feedback leadership team
	- Reviews
	- Documentation
	- Looms & Recordings
	- Presentations & Demos
- Receive and give constructive feedback based on work
- Step into leadership roles as needed (take over)
- Participate in the design process and identify the cost of implementation and workarounds
- 1v1 Mentorship with the leadership team 

# Business Highlights
**Our founders are experts in 3D and advertising**  
Founders are ex leadership of leading advertising and VR agencies in Los Angeles, doing work for Marvel, Disney, Riot Games, and more

**We’ve found product-market fit with  $14Ok+ in revenue**  
User base spans single product start-ups, multi-product businesses, and agencies.

**Our target market is $4B+ and growing**  
Every business selling a product online needs video  
to stand out and convert.

# If this sounds like a great fit
I’m on the founding team of Glossi ([glossi.io](http://glossi.io/)), a start-up using real-time rendering to create a cloud-based photo/video studio. Our software is built for entrepreneurs to e-commerce brands, blending game engine tech (unreal engine) with a native web experience to create a new category of product. We currently have a solid stable of customers - Epson, Playstation, Marshall, and more - with many more in the pipeline.

Please reach out to me: 
thomas.underwood@glossi.io

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for June is up (https://github.com/vercel/next.js/discussions/37379) and you're more than welcome to repost this message there!

**Comment by timlisiecki** [comment]
[Snyk](https://snyk.io/) | Frontend Engineer | Remote (US) | Full-time | [Link](https://boards.greenhouse.io/embed/job_app?token=5663456002&gh_src=3f9b65652us)
For this role, you will work as part of our Web Team within the Global Marketing Team to build a modern, static-generated site using Next.js and a headless CMS. You will partner with other developers on the development, optimization, and growth of our marketing websites.
I've been at the company for 6 months and love the team I'm working in. Our current website is Wordpress, but we are in the process of rebuilding it using a headless CMS with a Next.js frontend. Our team is in the Marketing Department, but we are made up of engineers and UX designers. Snyk is also[ growing rapidly](https://snyk.io/news/snyk-closes-530-million-series-f-investment-at-8-5-billion-valuation/), so it's a great time to join the team! Feel free to apply using the link above.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for May is up (https://github.com/vercel/next.js/discussions/36738) and you're more than welcome to repost this message there :)

**Comment by gregrickaby** [comment]
[WebDevStudios](https://webdevstudios.com/)  | Engineers (All levels) | Remote | Full-time / Part-time | $80-$100k Base | [Link](https://webdevstudios.com/careers/)

### About

WDS is one of the oldest WordPress agencies out there and a [VIP Gold Partner.](https://wpvip.com/) Lately, we've been shipping headless websites powered by WordPress and Next.js. The engineering department is inclusive, supportive, and fosters an environment of growth. 

**Benefits**

- Work from anywhere, in any time zone
- Generous PTO: 15 days per year + all US Federal Holidays
- 5 days paid sick leave per year
- Flexible work schedule
- 100% paid medical for single employees (US only)
- 401K + employer match (US only)
- Continuing education opportunities via LinkedIn Learning
- Parental leave
- Deliverables-based time tracking

### The Role

You'll be collaborating with an amazing project team lead by a Project Manager and Lead Engineer as they partner with enterprise level clients like [Microsoft](https://www.microsoft.com/en-us/security/business/security-insider/), [Monster.com](https://hiring.monster.com/resources/blog/), and the [NBA](https://thebal.com) to build and maintain websites. Applicants should be comfortable with WordPress, Next.js and consuming data via GraphQL and REST-API's.

You'll also have monthly 1:1's with the Engineering Manager and Employee Experience Coordinator, Quarterly-Check-ins, the opportunity to hang out on our weekly engineering scrums, plus paid training opportunities. 

If you're interested, check out our careers page! If you have any questions, feel free email me at: greg@webdevstudios.com

Thank you! 🍻

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for May is up (https://github.com/vercel/next.js/discussions/36738) and you're more than welcome to repost this message there :)

**Comment by YuriGii** [comment]
[RMRK](https://www.rmrk.app) | Front-end/Full stack engineers | Remote | Full-time | Competitive salary plus bonuses | [Link](https://github.com/rmrk-team/rmrk-web/wiki/RMRK-Jobs)

RMRK is a next-generation NFT protocol of composable, evolvable, forward-compatible, multi-chain NFTs. Founded in August of 2020, the team was formally created in February 2021 and has released several platforms and products since. Notably, [Singular](https://singular.rmrk.app/), an NFT marketplace and [Kanaria](https://kanaria.rmrk.app/) an RMRK 2.0 product demo of RMRK 2.0 technology, together facilitate over 3 million USD in volume per month.

RMRK's goal is to build the infrastructure to build awesome UX-centric things. Thus, the RMRK Association is the steward and main developer of the open-source RMRK specification, and its chief user.

Ultimately, we want to see NFTs go far beyond static media, and RMRK is about making that a reality. Come re-learn everything you know about NFTs with us!

learn more about RMRK [here](https://url.rmrk.app/rmrkcc) (video) or [here](https://url.rmrk.app/dawn) (text)
learn more about Kanaria [here](https://url.rmrk.app/kanariacc) (video) or [here](https://url.rmrk.app/demobird) (text)
learn more about $RMRK tokens [here](https://singular.rmrk.app/tokens)[](https://github.com/rmrk-team/rmrk-web/wiki/Full-Stack-React-developer#about-the-job)[](https://github.com/rmrk-team/rmrk-web/wiki/Full-Stack-React-developer#about-you)[](https://github.com/rmrk-team/rmrk-web/wiki/Full-Stack-React-developer#additional-info)
About the job
We are looking for a Full-Stack Typescript developer, with strong front-end skills. Namely React/Typescript/Next.js

About you
Ideally, you will have...

4+ years experience working with Typescript and React.
Experience working with Next.js and Node.js.
Experience in building high-performance web applications.
CSS best practices and CSS in JS approach (Chakra-UI, Fela, Glamor, etc).
Excellent communication skills, solid work ethic, and a strong desire to write efficient, scalable, and clean code.
Experience working in a high-paced environment.
Nice to have

Graphql/Apollo
PostgreSQL
Familiarity with Polkadot stack (Substrate, Polkadot blockchain)
Polkadot JS API
Solidity / Rust / Ink!
Additional info

Apply via [jobs@rmrk.app](mailto:jobs@rmrk.app)

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for May is up (https://github.com/vercel/next.js/discussions/36738) and you're more than welcome to repost this message there :)

**Comment by bcdunbar** [comment]
[Frontier](https://www.frontieradvisors.com.au/technology/) | UX / Front-end Developer | Hybrid ( Remote & Melbourne, Australia) | Full-time | Competitive salary

As Australia’s leading independent asset consultant, Frontier has been at the forefront of delivering investment advice and technology to institutional investors for more than 25 years. A key part of this is our Frontier Partners Platform - a multi-tenant integrated research and analytics platform, that allows institutional investors to explore new insights and achieve better investment outcomes. 

Currently, we are looking for a UX / Front-end Developer who can design and build compelling user experiences and lift Frontier Partners Platform to the next level. More specifically, the ideal candidate would have experience in design tools (Figma, Sketch, etc.), component building (Storybook) / styling (Styled Components or Emotion), and Next.js plus experience in state management (Context API or Redux, etc.), caching, and queries (fetch, react-query, etc.). An understanding of advanced methods such as multi-zones (or Module Federation), monorepos (Turborepo or Nx) and measuring performance would be useful.  

If you'd like to find out more, [message me via LinkedIn](https://www.linkedin.com/in/brandendunbar/)

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for May is up (https://github.com/vercel/next.js/discussions/36738) and you're more than welcome to repost this message there :)

**Comment by mj406** [comment]
> # What is this?

> > WWW.CROSSMINT.IO | ENGINEERING MANAGER | MIAMI FL/ HYBRID | Full-time | $100-178k base | LINKs
> 
> Responsible for hiring, managing, and deploying resources across several teams focused on developing web3 technology.  Will coach and mentor younger engineers, as well as review code submissions. Helps create a growth-minded work environment with a team motivated to execute quickly and deliver quality. Understands tools, architecture, security and scalability needs for web products.

Tech stack:
Next.js
TypeScript
Ethers.js
Solana (web3.js)
MongoDB
AWS



> 
> If you're a developer looking to post about themselves, use #34948.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for May is up (https://github.com/vercel/next.js/discussions/36738) and you're more than welcome to repost this message there :)

**Comment by omure** [comment]
Omure | Senior React.js Engineer | Remote (Europe, Americas) | Full-time | [Link](http://omure.com/jobs/senior-react)

We're looking for experienced React/Next devs to join our Front-end team.
There's no legacy or client work involved (we incubate startups) and hours are flexible.

Currently we offer a lot of room to grow fast - within the next year.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for May is up (https://github.com/vercel/next.js/discussions/36738) and you're more than welcome to repost this message there :)

**Comment by tuonglam95** [comment]
Leflair - SoPa | Senior Data Engineer | Onsite (Manila, Philippines) | Full-time

We are looking for Senior Data Engineer for Leflair - just been acquired by SoPa (Society Pass), to help build data warehouse. 
If you're interest, DM me via [Linkedin](https://www.linkedin.com/in/quynhvo39/) , I'm more than happy to discuss.
Have a nice day!

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for May is up (https://github.com/vercel/next.js/discussions/36738) and you're more than welcome to repost this message there :)

**Comment by seiflotfy** [comment]
### **[Axiom](https://axiom.co)** | Next.js/Typescript Engineer | Remote - US (Eastern), Europe, United Kingdom) | Full-time | [Apply](https://www.axiom.co/careers/5977017002)

**About Axiom**

Axiom’s mission is to empower developers to get the best insights into their data, as fast as possible. We are a remote-first and globally distributed team building a cloud native, serverless data analytics platform. Axiom completely changes the way in which developers and organizations think about their data: they can now send unlimited data with cost-effective storage and lightning-fast querying.

**About the Role**

We are seeking a Next.js/Typescript engineer to join our growing team to build an incredible observability experience for Next.js users. You will be designing and developing open source libraries that provide a fully integrated experience for Next.js users and Axiom.

**What You Will do**

- Design and implement robust and testable Typescript libraries
- Work closely with product and design teams to define the scope and scale of upcoming features
- Develop and maintain open source libraries
- Engage with Next.js developers from across the globe
- Troubleshoot, bug fix, and improve existing code
- Identify bottlenecks and increase the efficiency of software
- Communicate technical design decisions with the wider team
- Publish and maintain packages on npmjs

**Must-Have Qualifications** 

- 3+ years experience with Javascript, Typescript, and React
- Experience of the Next.js ecosystem
- Experience building complex libraries
- Experience designing and communicating technical architecture
- Experience with debugging and profiling tools for javascript code

**Bonus Qualifications**

- Experience building Next.js/React libraries in open source
- Experience publishing and maintaining public javascript packages
- Experience with observability/monitoring tools
- Experience working with globally distributed and remote teams


Feel free to [apply here](https://www.axiom.co/careers/5977017002)!

For further questions please reach out to me:
[seif@axiom.co](mailto:seif@axiom.co)

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for June is up (https://github.com/vercel/next.js/discussions/37379) and you're more than welcome to repost this message there!

**Comment by dan-made** [comment]
[Made In](https://madeincookware.com) | Senior Full-Stack Engineer | Remote (Continental US) | Full-Time | Competitive Salary + Benefits | [Apply Here](https://www.linkedin.com/jobs/view/2967860036/)

Made In is the leader in the digitally-native kitchen space. We bring Chef expertise and centuries-old cookware manufacturing techniques to craft professional-quality kitchen tools for the home and professional cook. We are backed by some of the best chefs in the world and have been featured in publications and shows such as the TODAY Show, NY Times, Fast Company, Business Insider, and more.

We’re seeking a motivated Senior Full Stack Engineer who is committed to helping us build a best-in-class eCommerce experience. In 2021 we moved to a cutting-edge headless architecture, and this year we have big plans to scale and enhance our new fully-owned NextJS frontend. We’re looking for generalist web developers with deep roots in the browser, who thrive in an atmosphere that’s nimble and ambitious. If you have natural initiative, curiosity, some hustle, and a sense of humor, then you might be a great fit at Made In.

In this role you will work regularly with our product team and designer, as well as fellow engineers. You’ll be the second most-senior technical resource on the team, and will report directly to our Lead Engineer.

## Responsibilities:

* First and foremost, build and ship user-facing features on our core NextJS eCommerce site
* Contribute to backend / infra projects to support the frontend
* Provide ongoing code review and mentorship to other engineers
* Collaborate with product / design resources on project status, concerns, and roadblocks
* Surface bugs, elevate best practices, and opportunities for improvement as they appear
* Contribute to the growing engineering team culture at Made In

## About You:

* You have 4+ years in frontend or full stack web development
* You can demonstrate career progression to a senior level of contribution
* You have a broad functional understanding of web fundamentals such as Javascript development, browser APIs, debugging and dev tools, CSS, performance, etc.
* You are able to communicate and collaborate effectively – with technical and non-technical parties
* You can operate at different zoom levels – can think holistically about the big picture, and can also attend to fine-grained implementation details
* You are interested and able to function in a fast-moving startup environment – you can wear multiple hats and adapt to roadmap changes

## Preferred (Bonus) Experience:

* Server-side rendering technologies e.g. Gatsby, Next, etc.
* Ecommerce platforms, specifically Shopify
* Content Management Systems like Sanity or Contentful
* Google Tag Manager and third party data platforms

## Benefits:

- Remote eligible*
- Competitive salary
- Equity grant
- Healthcare, Dental, and Vision
- Flexible vacation policy
- Generous employee discount

*Made In headquarters is based in Austin, TX. This position is remote eligible within the continental United States.

Must be eligible to work in the United States. We are unable to provide Visa sponsorship for employment at this time.

Made In is an equal opportunity employer. We celebrate diversity and are committed to creating an inclusive environment for all employees.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for June is up (https://github.com/vercel/next.js/discussions/37379) and you're more than welcome to repost this message there!

**Comment by dlindenkreuz** [comment]
### [Ren](https://rensystems.com) | Full Stack Engineer (Frontend focus) | Remote - EU timezones | Full-time | [Apply here](https://rensystems.notion.site/Full-Stack-Engineer-Frontend-694747ad002143b38f3a89a6076b75dc)

### Who we are

> ren ( 仁 ): The Confucian virtue denoting the quality of being altruistic

Ren is on a mission to help people strengthen and leverage relationships to achieve their goals and create more value. We're an early-stage startup with hubs in New York and Zurich. We raised a $5m Seed round from an incredible group of founders, leaders and investors.

Using cutting edge Natural Language Processing and Deep Learning technology, Ren helps leaders, deal-makers and chiefs of staff stay on top of important moments in their networks, and makes it fast, easy, and personal to reach out with the right context.

### About the role

As a full stack engineer with a user centric focus, you will work on the frontend development of our product, reporting directly to the CTO. In a team of full stack engineers, machine learning experts and designers, you will be responsible for building our web application.

### Frontend Tech Stack

React, Typescript, Next.js, Chakra UI

---

**My personal opinion:** We are a close-knit team of passionate people who love their craft. We have a compelling, well-made product and plenty of excited users. Ever since I joined Ren in 2020, I've never looked back.

[➡️ **More Information & Apply Here**](https://rensystems.notion.site/Full-Stack-Engineer-Frontend-694747ad002143b38f3a89a6076b75dc)

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for June is up (https://github.com/vercel/next.js/discussions/37379) and you're more than welcome to repost this message there!

**Comment by mark-tomlinson-dev** [comment]
[Aeon Media Group](https://aeon.co/) | Lead Frontend Developer | Remote (Australia) | Full-time | $110-130k AUD | [Apply](https://www.linkedin.com/jobs/view/2975649185/?refId=pdhZ2bIRX8p5VSwlR%2FRkXA%3D%3D&trackingId=bKJss1SHbV1swGqIbSoDqg%3D%3D)

Aeon Media Group is a non-profit organization and registered charity, and the publisher of [Aeon](https://aeon.co/) and [Psyche](https://psyche.co/) magazines. We are looking for an experienced frontend developer to join our growing global team. You should be based in Australia or in a similar time zone.

I've been at the company for 6 months and and love it so far. The stack is nice (Next, GraphQL, Rails, Docker, Kubernetes) and the organization is doing interesting work. Feel free to apply using the link above or reach out to tech@aeon.co.

[**More info**](https://www.linkedin.com/jobs/view/2975649185/?refId=pdhZ2bIRX8p5VSwlR%2FRkXA%3D%3D&trackingId=bKJss1SHbV1swGqIbSoDqg%3D%3D)

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for May is up (https://github.com/vercel/next.js/discussions/36738) and you're more than welcome to repost this message there :)

**Comment by sullivan-sean** [comment]
Hey everyone! 

I'm Sean, founding engineer @ Fig ([fig.io](http://fig.io)). Fig makes developers more productive in the terminal. 

- **Mission**: re-imagine the terminal. First, autocomplete, then a fully-fledged App Ecosystem 
- **Why**: 50M+ engineers use the terminal. It has barely changed since the 70s.
- **Investors**: Y Combinator, General Catalyst, Kleiner Perkins, founders/execs at Stripe, GitHub, Heroku, Segment
- **Traction**: One of the fastest growing OSS startups in 2021; our users love us: see twitter.com/fig

We're looking for frontend engineers experienced with Next & React to come join us and help lead our new product 🤫

If you're interested, please reach out to me (sean@fig.io) or apply at https://fig.io/jobs/frontend

**Reply by Nutlope** [reply]
Hey Sean! Just want to let you know that the new job board for May is up (https://github.com/vercel/next.js/discussions/36738) and you're more than welcome to repost this message there :)

**Comment by VegFer** [comment]
Housecall Pro | Team Lead Engr. React | Fully-remote | Full-time | $102-132k base | [Apply here](https://bit.ly/3F5o6mf)

What you'll be doing

Design, develop and deploy features for Housecall Pro’s core web application in React/Redux
Ensure the performance, quality, and responsiveness of applications
Work with product management to refine initial concepts features that deliver customer delight
Build and execute a high velocity of smart, highly performant server-side A/B tests to increase business metrics
Understand clear sub-system boundaries, and make recommendations on where functionality belongs across different sub-systems
Stay up to date with new Growth and Personalization tools
Identify and correct browser compatibility problems and ensure UX designs are properly translated to responsive solutions across a wide range of device resolutions
Develop and maintain code quality standards and enforce them across the team
Continually introduce architectural improvements and refactorings to the code baseline
Write automated tests at the appropriate architectural level
Conduct code reviews and architectural reviews for features developed by other members of the team
Lead feature teams in their weekly iterations
Maintain and improve our continuous integration and development environments


Core qualifications

Ability to work at a fast pace with a drive to quickly deliver value to our customers
4 - 8 years industry experience developing commercial React applications
2 - 4 years experience leading teams and performing code reviews
Advanced knowledge of Javascript
Attention to detail and an eye for design
Can communicate and work independently with non-technical stakeholders to see through large epics
Data-Driven
Depth of understanding and teaching ability in clean code, refactoring, and test-driven development
Familiarity with Restful APIs used to connect React applications to back-end services
Strong understanding of underlying systems architecture
Independently gain knowledge and opinions on multiple systems
Researches and assesses current and new technologies
Commercial experience with Ruby/Ruby on Rails a plus
Experience with MaterialUI a plus
Fluent Polish and English language

**Comment by danielsdesk** [comment]
ocV!BE | Sr Product Engineer - UI/Web | Remote (US/CAN) | Full-time | Permanent

Hi, we're hiring for a Senior Software Product Engineer UI/Web. React.js/Next.js and more is applicable experience.

[Apply Here](https://anaheimducks.wd5.myworkdayjobs.com/en-US/HC/job/Senior-Software-Product-Engineer--UI-Web_R-2022-183-2)

The Work:  As a UI/Web Senior Software Product Engineer, you will also work with a multidisciplinary team such as System Engineers, Backend Engineers, and Creative Technologists—to help guide product visions to reality by:

- Being responsible for designing and implementing intelligent and efficient UIs, spinning up and maintaining robust UI components, and developing speedy and modular content management systems by developing and delivering impactful code that enables the success of Litehouse user applications and embedded webviews
- Implementing modern web application stacks as well as executing user interface designs for a variety of product applications, experiential applications, and UI tools in potentially non-traditional ways
- Participating in prototyping, testing, developing, deploying, maintaining, and evolving items on the Litehouse product roadmap
- Contributing to exceptional software, hardware, and design solutions that exceed expectations
- Deeply understanding our product in support of its development, growth, and enablement of user success
- Contributing to exceptional software, hardware, and design solutions that exceed expectations
- Writing clean, end-to-end code in support of the above
- Understanding the bigger picture and the purpose of software to ensure and improve its quality and fitness-for-purpose


Skills

- Strong understanding of Modern UI architectures, React, and JavaScript, with some experience with Next.js and Webpack
- Responsive UI Development w/ HTML, CSS, and related web technologies
- Expertise in integrating with APIs (REST and GraphQL w/ Relay) and Socket communication (Websocket/STOMP)
- Expertise in advanced UI State Management techniques for React with libraries such as Redux, XState, and Jotai
- Ability to participate within Agile Component-Driven Development and UI Review/Prototyping processes
- Someone who thrives in novel, first-of-a-kind settings, with creative contribution informed by technical acumen
- An excellent independent problem solver and proactive communicator who can self-direct
- Ability to learn and master new technologies very quickly


Culture & Fit

This role is an excellent fit for anyone who:

- Is an independent, careful thinker who is passionate about the work and places a premium on craftsmanship, professional responsibility, data privacy, and ethical design
- Is excited to work on bleeding-edge technologies and devote mindshare to solving the problems of tomorrow, today
- Has interest in microfrontends via Module Federation, and UI Review with Storybook
- Has interest in Adaptive/Intelligent UIs and workflows
- Has interest in 3D technologies such as WebGL/Three.js/React-Three-Fiber for AR/VR/XR, and/or gaming, as well as other audio/video media technologies such as music production or event production
- Has interest in WebAssembly for Web, Desktop, and Mobile Apps, and potential languages for it such as Rust/Assemblyscript
- Has interest in creative technology, UX, workflow management, systems thinking, art, architecture, smart cities, and AI
- A healthy appreciation for and use of Slack’s Giphy integration
- Lives in or might be interested in relocating to sunny Southern California

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for May is up (https://github.com/vercel/next.js/discussions/36738) and you're more than welcome to repost this message there :)
